lib/layer/Style.mjs

```diff
@@ -135,20 +188,4 @@ export default layer => feature => {
     // Delere style.label if current zoom exceeds maxZoom.
     feature.style.label?.maxZoom < layer.mapview.Map.getView().getZoom() && delete feature.style.label;
   }
-
-  // Check whether the feature referenced in mapview.locations
-  if (layer.mapview?.locations[`${layer.key}!${feature.properties.id}`]) {
-
-    // The feature geometry should not be displayed.
-    if (layer.style.selected === null) return;
-
-    if (layer.style.selected) {
-
-      mapp.utils.merge(feature.style, layer.style.selected, {
-        zIndex: Infinity,
```

Comment:
is the zIndex required on grid layers?

<img width="806" alt="image" src="https://github.com/GEOLYTIX/xyz/assets/43169158/3fe1a583-71b3-41f8-b5f8-6fa03104da00">

Reply:
No it's not required as the grid layer will be removed https://github.com/GEOLYTIX/xyz/pull/871

Reply:
Gotcha!